### PR TITLE
minor performance enhancement for OMC API

### PR DESCRIPTION
**Pull request by adrpo**

### CODE REVIEW

**Comment by OpenModelica-Hudson** [comment]
The test suite is unstable according to [OpenModelica_TEST_PULL_REQUEST 2016-12-08_10-35-28](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3517/).

File: Compiler/Script/Interactive.mo [hunk 1]
```diff
@@ -3946,6 +3946,74 @@ algorithm
...
+  ocache := getGlobalRoot(Global.interactiveCache);
+  if isSome(ocache) then
+    SOME(cache) := ocache;
+    for x in cache loop
+      (po, patho, envo) := x;
+      // found the path
+      if Absyn.pathEqual(patho, p_class) then
+        // make sure the program did not change
+        if referenceEq(po, p) then
+          print("Got it from cache: " + Absyn.pathString(patho) + "\n");
```
Comment (sjoelund) [inline Compiler/Script/Interactive.mo]: Debug print shouldn't be here (there are more of them)

**Comment by OpenModelica-Hudson** [comment]
The tests run correctly according to [OpenModelica_TEST_PULL_REQUEST 2016-12-08_11-52-58](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3519/console).

**Comment by casella** [comment]
Can you push this change to 1.11.0 Beta also?

**Comment by adrpo** [comment]
Of course, that is the idea. I will probably need some help from Martin to cherry pick some commits from the master branch to push in the 1.11 maintenance as is currently not building on Windows after the latest commits.

**Comment by casella** [comment]
Good luck then :)

**Pull request closed without merging**